sdk/keyvault/azure-keyvault-administration/samples/backup_restore_operations.py

```diff
@@ -0,0 +1,62 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+import os
+
+from azure.keyvault.administration import KeyVaultBackupClient
+from azure.identity import DefaultAzureCredential
+from azure.core.exceptions import HttpResponseError
+
+# ----------------------------------------------------------------------------------------------------------
+# Prerequisites:
+# 1. A managed HSM (https://docs.microsoft.com/azure/key-vault/managed-hsm/quick-create-cli)
+#
+# 2. azure-keyvault-administration and azure-identity libraries (pip install these)
+#
+# 3. Set Environment variables AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET, VAULT_URL
+#    (See https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/keyvault/azure-keyvault-administration#authenticate-the-client)
+#
+# 4. A storage account containing a blob storage container
+#    (See https://docs.microsoft.com/azure/storage/blobs/storage-blobs-introduction)
+#
+# 5. Set Environment variables CONTAINER_URL and SAS_TOKEN corresponding to your blob container's URI and SAS
+#    (See https://docs.microsoft.com/azure/storage/common/storage-sas-overview)
+#
+# For more details, see https://docs.microsoft.com/azure/key-vault/managed-hsm/backup-restore
+#
+# ----------------------------------------------------------------------------------------------------------
+# Sample - demonstrates vault backup and restore operations for Managed HSM
```

Comment:
Maybe it's clearer (or at least less likely to promote confusion) to consistently refer to these operations as "full backup" and "full restore", i.e. avoid using "vault" for fear of implying Key Vault supports them?